PrivateCollections/Germany/Müth001.xml

```diff
@@ -0,0 +1,209 @@
+                    <summary>Some text is partially visible at the rands of miniatures.
+                      Apparently, the miniatures are executed over the text at a later stage, probably in order
+                    to increase a market value of the manuscript.</summary>
+                    <decoNote type="miniature" xml:id="d1">
+                      <locus target="2v"/>
+                      <desc>Miniature
+                        of <ref type="authFile" corresp="AT1012Equestrian"></ref>,
+                      namely of<persName ref="PRS4585Georgeo">St George</persName>,
+                   the <term key="dragon">dragon</term>, and the <term key="angel"></term>.</desc>
+                    </decoNote>
+                    <decoNote type="miniature" xml:id="d2">
+                      <locus target="#6r"/>
+                      <desc>
+                      Miniature of <ref type="authFile" corresp="AT1031VirginAndChild"></ref>,
+                      and two
+                      <term key="angel">angels</term> with <term key="sword">swords</term>.</desc>
+                    </decoNote>
+                    <decoNote xml:id="d3" type="miniature">
+                      <locus target="#27r"></locus>
+                      <desc>Miniature of
+                        <ref type="authFile" corresp="AT1005Crucifixion"></ref>, depicting
+                      also <persName ref="PRS6819Mary"></persName> and
+                    <persName ref="PRS5695John"></persName>.</desc>
+                    </decoNote>
+                    <decoNote xml:id="d4" type="miniature">
+                      <locus target="#35r"></locus>
+                      <desc>Miniature of the <ref type="authFile" corresp="AT1050ApparitionResur"></ref>.</desc>
+                    </decoNote>
+                  </decoDesc>
+                  <additions>
+                    <list>
+                      <item xml:id="e1">
+                        <locus target="#16v #17v"></locus>
+                        <desc>Crude note or addition to the text in pen.</desc>
+                      </item>
+                    </list>
+                  </additions>
+                  <bindingDesc>
+                     <binding contemporary="true" xml:id="binding">
+                        <decoNote xml:id="b1">Two wooden boards covered with brown blind-tooled leather.</decoNote>
+                        <decoNote xml:id="b2" type="Boards">Inlays of reused parchment with unidentified texts. The inlay
+                        of the bottom board has stains of pink and violet color.</decoNote>
+                        <decoNote xml:id="b3" type="bindingMaterial">
+                          <material key="wood"/>
+                          <material key="leather"/>
+                          <material key="parchment"/>
+                        </decoNote>
+                        <decoNote xml:id="b4" type="SewingStations">Two pairs of sewing stations.</decoNote>
```

Comment:
4, according to our guidelines

Reply:
sorry, I probably do not understand what you mean. According to the guidelines we have to document "pairs".

Reply:
There must be a misunderstanding, which means probably that after checking we should rephrase the guidelines. I understand the paragraph "sewing stations" in https://betamasaheft.eu/Guidelines/?id=bindingDescription so that the number should be encoded in the way I did here. But now I wonder whether that should be done in addition to another element with type SewingStations with the more traditional "pairs" phrasing? Maybe @DenisNosnitsin1970 or @ElianaDalSasso can help us.

Reply:
Ok, I see, yes there is a confusion since Example 1 on that page in GL has a different way of encoding. It would be good to clarify this and to document the decision in the GL.

Reply:
I have corrected the coding! It would be nice to merge it so I can go for a PR with new shelfmarks